flink-next-gen-connector-influxdb/src/main/java/org/apache/flink/streaming/connectors/influxdb/sink/commiter/InfluxDBCommitter.java

```diff
@@ -42,23 +40,30 @@ public InfluxDBCommitter(final InfluxDBConfig config) {
     // This method is called only when a checkpoint is set
     @SneakyThrows
     @Override
-    public List<Void> commit(final List<Void> committables) throws IOException {
+    public List<Long> commit(final List<Long> committables) {
         log.info("A checkpoint is set.");
-        this.writeCheckpointDataPoint();
+        long lastTimestamp = 0;
```

Comment:
Why init with 0? Rather null right? So influx would use its own timestamp and not 0